src/lib/Components/Contact.svelte

```diff
@@ -0,0 +1,83 @@
+<script>
```

Comment:
Malo uso, nice work! Are you able to pull the latest changes from main and rebase off it on your branch please